builtin/builtin.go

```diff
@@ -443,7 +443,11 @@ var Builtins = []*Function{
 		Name: "fromJSON",
 		Func: func(args ...any) (any, error) {
 			var v any
-			err := json.Unmarshal([]byte(args[0].(string)), &v)
+			jsonStr := args[0]
+			if strPtr, ok := jsonStr.(*string); ok {
```

Comment:
Althoug this is a working solution, we need to understand why OpDeref was not added before calling fromJSON. 

Compiler adds OpDeref for arguments before calling a function. But only if needed. Aperently the check is not perfect.

Reply:
I figured out why. Builtins compilation just do not check for deref. So we ether make compiler more advance. Or make it via additional call.